### ISSUE TITLE
Unable to compile

### ISSUE THREAD

**Issue by Mrdarknezz** [PocketRent/hhvm-pgsql#55]
In file included from /usr/include/hphp/runtime/base/base-includes.h:24:0,
                 from /home/vagrant/hhvm-pgsql-master/pgsql.h:6,
                 from /home/vagrant/hhvm-pgsql-master/pgsql.cpp:1:
	/usr/include/hphp/runtime/ext/extension.h:26:34: fatal error: hphp/runtime/version.h: No such file or directory
	 #include "hphp/runtime/version.h"
                                  ^
	compilation terminated.
	make[2]: *** [CMakeFiles/pgsql.dir/pgsql.cpp.o] Error 1
	make[1]: *** [CMakeFiles/pgsql.dir/all] Error 2
	make: *** [all] Error 2

This is what I get when I try to compile.

**Comment by jaredkipe** [comment]
I ran into the same issue in homestead.
HHVM was ~3.4.1

The referenced file,  /usr/include/hphp/runtime/ext/extension.h, did have that include in it.  The file was missing, and the 3.4 branch of hhvm does not have the same reference/include.

Commenting out the #include hphp/runtime/version.h WILL allow you to compile
....BUT you cannot load the result .so due to missing symbols.

To fix.  
apt-get update && apt-get upgrade hhvm hhvm-dev -y

Delete CMakeCache.txt, make clean, etc and start the build process from scratch.
Bonus, you're running HHVM ~3.5 and I got a new version of Postgres at the same time (9.4.1)
